File: FoodRegistry/src/MenuItem.cpp
#include "MenuItem.h"
#include <iostream>
#include <iomanip>
using namespace std;

MenuItem::MenuItem(char c, string n, float p) : code(c), name(n), price(p) {}

char MenuItem::getCode() const { 
    return code; 
}

string MenuItem::getName() const { 
    return name; 
}

float MenuItem::getPrice() const { 
    return price; 
}

void MenuItem::display() const {
    cout << '(' << code << ") " << name << endl;
    cout << "Price: RM " << fixed << setprecision(2) << price << endl;
}

File: java_src/com/foodregistry/MenuItem.java
package com.foodregistry;

public class MenuItem {
    private char code;
    private String name;
    private float price;

    public MenuItem(char c, String n, float p) {
        this.code = c;
        this.name = n;
        this.price = p;
    }

    public char getCode() {
        return code;
    }

    public String getName() {
        return name;
    }

    public float getPrice() {
        return price;
    }
}
